### ISSUE TITLE
[Request] Options

### ISSUE THREAD

**Issue by Gelo3D** [stanac/shutdown#3]
Add new timing option : 

locked pc - example - if You download something and u wonna protect Your pc u need to example 10/15 minute to lock Your computer

sleep pc - sleep pc is best option if u need fast rest Your pc to work.

**Comment by stanac** [comment]
Hi @Gelo3D sleep option does make sense, but I would disagree on lock, What is the use case of delaying lock operation? From your comment about lock... I don't see what is stopping your PC from downloading something while PC is locked?

**Comment by Gelo3D** [comment]
Dat was just only example. 
Just do very simple lock pc option. 
Sry bro for my english, that was a reasion for had trouble understanding.

**Comment by stanac** [comment]
NP, I will investigate sleep mode

**Comment by zvonimirr** [comment]
Is this still open and being considered?
I would like to contribute.

**Comment by stanac** [comment]
Hello @rudinskizvonimir ticket is still open. Contribution would be welcome. Thank you.

**Comment by zvonimirr** [comment]
Great, I'll start as soon as I can

**Comment by theogjpeezy** [comment]
I have a solution for this.  The existing shutdown command in the project does not actually support a sleep.  There is a way to use rundll32.exe to set the power profile to force sleep, however, if the OS has hibernation enabled then it will cause the machine to go into hibernation, not sleep. [Described Here](https://superuser.com/questions/39584/what-is-the-command-to-use-to-put-your-computer-to-sleep-not-hibernate#answer-39652)
The solution seems to be using [PsShutdown](https://technet.microsoft.com/en-us/sysinternals/psshutdown.aspx) in the [PsTools Suite](https://download.sysinternals.com/files/PSTools.zip).  After including this in the project it works as expected.  To minimize scope, I have only implemented using PsShutdown for sleep, all other features still utilize shutdown.exe.  The only (minor) drawback is that the user will have to agree to the SysInternals EULA when they select sleep for the first time.  After accepting the EULA, they will not see this dialog box again.
If this approach sounds appropriate, I can submit a Pull Request for review.

**Comment by stanac** [comment]
Did you try [Application.SetSuspendState](https://msdn.microsoft.com/en-us/library/system.windows.forms.application.setsuspendstate(v=vs.110).aspx) method?

Simple test on my laptop showed that both hibernate (already implemented) and sleep (also called suspend, with mentioned method) worked.

We don't want to complicate and introduce unneeded dependencies. We should use `SetSuspendState` and if it's not working on some PCs we can leave a comment on [the web](http://stanac.github.io/shutdown/) why it's not working and that it won't be handled for all edge cases.

Here is the code I've tried:
```cs
System.Windows.Forms.Application.SetSuspendState(
    System.Windows.Forms.PowerState.Suspend,
    force: false, 
    disableWakeEvent: false);
```

We just need "force" option to be set in there.

**Comment by stanac** [comment]
@rudinskizvonimir please let me know what you think about my previous comment.

**Comment by theogjpeezy** [comment]
@stanac I'll give it a go.  I'll report back later today.

**Comment by theogjpeezy** [comment]
@stanac The SetSuspendedState method worked without issue.  I have reworked it so that it uses this.  If it is okay, I'd be ready to make a pull request at this time.

**Comment by stanac** [comment]
@theogjpeezy please create pull request. Thanks

**Comment by stanac** [comment]
implemented in #4
